storage/innobase/btr/btr0scrub.cc

```diff
@@ -216,17 +202,16 @@ check_scrub_setting(
 
 #define IBUF_INDEX_ID (DICT_IBUF_ID_MIN + IBUF_SPACE_ID)
 
-/**************************************************************//**
-Check if a page needs scrubbing */
-
+/** Check if a page needs scrubbing 
+@param[in]	scrub_data
+@param[in]	block		block to check, latched
+@param[in]	allocated	is block allocated, free, or unknown
```

Comment:
Comment changed from original, hopefully OK

Reply:
it is, unknown was mentioned in the header but not in the cc file